pkg/distro/image_config_test.go

```diff
@@ -166,3 +166,71 @@ func TestImageConfigInheritFrom(t *testing.T) {
 		})
 	}
 }
+
+func TestImageConfiDNCConfigNone(t *testing.T) {
+	var expected []*osbuild.DNFConfigStageOptions
+	cnf := &ImageConfig{}
+	assert.Equal(t, expected, cnf.DNFConfigOptions("centos-9"))
+}
+
+func TestImageConfiDNCConfigSetReleaseVer(t *testing.T) {
+	cnf := &ImageConfig{
+		DNFConfig: &DNFConfig{
+			SetReleaseVerVar: true,
+		},
+	}
+	assert.Equal(t, []*osbuild.DNFConfigStageOptions{
+		{
+			Variables: []osbuild.DNFVariable{
+				{
+					Name:  "releasever",
+					Value: "centos-9",
```

Comment:
Nitpick: I know that these are just unit tests, but the `releasever` is ever set only to `X.Y` in RHEL images (and for centos it is `X-stream`). This mechanism allows you to pin the host repositories to a specific Extended Update Support (EUS) version of RHEL. Using `centos-9` may give one the wrong impression of its purpose.

Reply:
Thanks, fixed in the other PR